crates/db_views/site/src/api.rs

```diff
@@ -711,6 +713,8 @@ pub enum PostOrCommentOrPrivateMessage {
 /// Be careful with any changes to this struct, to avoid breaking changes which could prevent
 /// importing older backups.
 #[derive(Debug, Serialize, Deserialize, Clone, Default)]
+#[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
+#[cfg_attr(feature = "ts-rs", ts(optional_fields, export))]
```

Comment:
I added this to fix the errors, but judging by the code comment saying this isn't meant to be parsed by apps/clients, I should probably remove the TS derive from `ExportPromiseResponse` instead.

Reply:
Its pry fine to leave it, the exports are harmless, and front ends might want to use it even if it is unstable.

Reply:
Yes its fine, anyway we cant make changes here or it will break all existing backups.